superset-frontend/.eslintrc.js

```diff
@@ -72,19 +78,26 @@ module.exports = {
         ],
         'import/no-named-as-default': 0,
         'import/no-named-as-default-member': 0,
+        'import/no-useless-path-segments': 0,
         'import/prefer-default-export': 0,
         indent: 0,
         'jsx-a11y/anchor-has-content': 0,
-        'jsx-a11y/href-no-hash': 0,
```

Comment:
did this rule get removed from the airbnb defaults? or was it renamed to `anchor-is-valid`?

Reply:
It was replaced in version 6 of eslint-plugin-jsx-a11y by `anchor-is-valid` (https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/commit/a14617528cda23c6c88902ace1d083e4e13ac271)